aldryn_newsblog/models.py

```diff
@@ -137,7 +149,7 @@ class Article(TranslatedAutoSlugifyMixin,
     tags = TaggableManager(blank=True)
 
     related = SortedManyToManyField('self', verbose_name=_('related articles'),
-                                    blank=True)
+                                    blank=True, symmetrical=ARTICLE_RELATED_SYMMETRICAL)
```

Comment:
Hard to say what people really expect. But yeah, I think it is better with symmetrical=False as well. 👍  from me for that approach.